### PR TITLE
Feature/0003 migrate bs delete modal

### PR DESCRIPTION
**Pull request by giselalanciani**

### CODE REVIEW

**Comment by giselalanciani** [comment]
Own CR done

File: dist/authors/index.html [hunk 1]
```diff
@@ -46,6 +46,41 @@ <h1>Authors</h1>
         </td>
       </tr>
     </template>
+    <!-- Modal -->
+    <div
+      class="modal fade"
+      id="delete-modal"
+      tabindex="-1"
+      aria-labelledby="exampleModalLabel"
```
Comment (diegolaciar) [inline dist/authors/index.html]: Please. replace the ID of the h1 to something like ` id="delete-modal-label"` and change the references to the div aria-labelledby attribute

Do the change on all your HTML views.

**Review by diegolaciar — CHANGES_REQUESTED** [review]
Please check the comments

File: dist/authors/index.html (hunk 1 above)
Comment (giselalanciani) [reply dist/authors/index.html]: Done

**Review by diegolaciar — APPROVED** [review]

**Comment by diegolaciar** [comment]
@giselalanciani ,

Great job! 🚀 Ship IT !

**Pull request merged**